doc/api/cli.md

```diff
@@ -2126,6 +2168,14 @@ added: v0.1.3
 
 Print node's version.
 
+### `--warnings`
```

Comment:
I don't think we generally document no-op flags.

Reply:
I can remove